### ISSUE TITLE
Default value for "Use project config"

### ISSUE THREAD

**Issue by rugk** [jchouse/csscomb-atom#24]
The setting "Use project config" should have the value  `.csscomb.json` by default as this is the default config file csscomb.js [advertises](https://github.com/csscomb/csscomb.js#2-configure).

**Comment by jchouse** [comment]
Actually the first item as time goes - Use one of [predefined configs](https://github.com/csscomb/csscomb.js/tree/master/config) and .csscomb.json is the second point. [advertises](https://github.com/csscomb/csscomb.js#2-configure).

**Comment by rugk** [comment]
Yeah, but if a `.csscomb.json` file is found in the repo then it should actually *always* be preferred over any default configuration. That's why a default value for this would be good.

**Comment by jchouse** [comment]
OK, I understood you, is should not be optionally? may be:) My way is more flexible

**Comment by rugk** [comment]
It seems you did not understand me. It *should be* **optional**, but it *should* **also** *be* the **default**.

The only thing I want is that *after installation* the configuration looks like this:
![defaultconf](https://cloud.githubusercontent.com/assets/11966684/16492292/94545df2-3ee1-11e6-8dba-442a597ef4b9.png)

A user who really wants to ignore the `.csscomb.json` file or wants to use another filename can change the value anyway afterwards. What matters here is the default value.

**Comment by jchouse** [comment]
I got it

**Comment by rugk** [comment]
Nice, so do you consider implementing this?

**Comment by jchouse** [comment]
I'll think about it
